### DISCUSSION TITLE
What engine Copilot uses now?

### DISCUSSION THREAD

**Discussion by CharlonTank** [community/community#13589] [Copilot Conversations]
What engine Copilot uses now?
`code-davinci-002` ?

Any idea when the `Insert` and `Edit` features we have on OpenAI Playground are gonna be available in Copilot?

**Comment by CharlonTank** [comment]
Up

**Comment by cashgarman** [comment]
Also eager to try out these new features. Their announcement blog post states: "This unlocks new use cases and improves existing ones; for example, insertion is already being piloted in GitHub Copilot with promising early results."

So I wonder if there is some nightly or preview build out there somewhere for one of the supported IDEs that does support it?

**Reply by CharlonTank** [reply]
Yeah, I would like to know too.

**Reply by cashgarman** [reply]
Honestly, with the announcement of VS getting Copilot yesterday, following that blog post so closely, I immediately thought "This is it, Microsoft is finally ready to blow away all other IDEs with Copilot VS integration with insert and edit".

Shame that doesn't seem to be the case :(

**Reply by CharlonTank** [reply]
Ok, I see, but I use a Mac, I cannot test because VS mac version is 17.0 and I need 17.1/17.2 to be able to download github copilot extension in VS.
